linkerd/proxy/transport/src/listen/dual_bind.rs

```diff
@@ -0,0 +1,79 @@
+use crate::{addrs::DualListenAddr, listen::Bind, Keepalive, ListenAddr};
+use futures::Stream;
+use linkerd_error::Result;
+use linkerd_stack::Param;
+use std::{net::SocketAddr, pin::Pin};
+use tokio::net::TcpStream;
+use tokio_stream::StreamExt;
+
+#[derive(Copy, Clone, Debug, Default)]
+pub struct DualBindWithOrigDst<B> {
```

Comment:
Nothing in this module cares about OrigDstAddr so we rename this

```suggestion
pub struct DualBind<B> {
```